src/autoinstrumentation.ts

```diff
@@ -61,7 +64,7 @@ const agentDescriptionIdentifyingAttributes = {
   [SEMRESATTRS_TELEMETRY_SDK_LANGUAGE]: TELEMETRYSDKLANGUAGEVALUES_NODEJS,
   [SEMRESATTRS_PROCESS_RUNTIME_VERSION]: process.versions.node,
   [SEMRESATTRS_TELEMETRY_DISTRO_VERSION]: VERSION,
-  [SEMRESATTRS_PROCESS_PID]: process.pid,
+  [PROCESS_VPID]: process.pid,
```

Comment:
If we already have the `OdigosProcessDetector`, perhaps we can populate this attribute there?

So in the same place we delete `process.pid` then add `process.vpid`.